src/wp-includes/class-wp-query.php

```diff
@@ -3270,10 +3270,11 @@ public function get_posts() {
 			return $post_parents;
 		}
 
+		$unfiltered_query = $old_request == $this->request && "{$wpdb->posts}.*" === $fields;
```

Comment:
Two things:
* Why a non-strict comparison here? Can't we use `===`?
* Nit-pick, but maybe `$is_unfiltered_query` is a better name here, to clarify that this is a boolean.

Reply:
I have changed the variable name. But I will not change the logic for now. That is the logic it was before and I don't want to change it.

Reply:
@spacedmonkey Why not use `===`? I see this was using `==` before, but I don't think there's a good reason not to fix it. The request is always a string, and I don't think for strings the `==` or `===` even makes a real difference. I think we should use `===` in accordance with the latest WordPress coding standards and using strict comparisons.